site/content/en/docs/tasks/run/external_workloads/appwrappers.md

```diff
@@ -1,6 +1,5 @@
 ---
 title: "Run An AppWrapper"
-linkTitle: "AppWrappers"
```

Comment:
No need to remove. Let's also keep AppWrapper where it was as it is a built-in framework currently. We have another page for showing how to use AppWrapper for custom Jobs.

Reply:
> No need to remove. Let's also keep AppWrapper where it was as it is a built-in framework currently. We have another page for showing how to use AppWrapper for custom Jobs.

I think this makes sense.  

Once Kueue 0.11 comes out, we plan to simplify the AppWrapper controller, remove the code that is no longer needed since it is now a built-in framework, and make a new release of AppWrapper. As part of doing that, I will revise the Kueue documention and make sure it refers to the right historic pieces of external-framework code in the AppWrapper 1.0 git.

Reply:
updated.